README.md

```diff
@@ -261,73 +259,350 @@ The SDK provides a simple interface for all the [media APIs mentioned here](http
 Accepts an object specifying the parameters to be used to list and search files. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/list-and-search-files) can be passed as it is with the correct values to get the results.
 
 ```cs
-List<ListAPIResponse> resp = await imagekit
-    .Skip(0)
-    .Limit(10)
-    .ListFilesAsync();
+ResponseMetaData resp = await imagekit.GetFileListRequestAsync();
```

Comment:
All our examples should include a few other options parameters supported by the method in addition to mandatory parameters. Please note that this feedback is for all examples throughout this readme and any other documentation related to this SDK or sample app.